packages/auth/src/model/public_types.ts

```diff
@@ -254,6 +254,16 @@ export interface Auth {
     error?: ErrorFn,
     completed?: CompleteFn
   ): Unsubscribe;
+  /**
```

Comment:
@egilmorez Can we get a review of the comments here? They will go into the reference docs.